artdaq-mu2e/ArtModules/DTCEventVerifier_module.cc

```diff
@@ -140,9 +151,30 @@ bool mu2e::DTCEventVerifier::filter(art::Event& event)
       
 	if ( subEvtWTag != evtWTag) { evtHeader->ewt_check = 0; } //different EWT in a subEvent
 	
+	if (metricMan != nullptr && diagLevel_ > 10)
+	  {
+	    std::ostringstream oss;
+	    oss << "SubEvent: "     << &subEvt - &data.GetSubEvents()[0] 
+		<< ", DTC_ID = "    << dtcID 
+		<< ", EWT_CHECK = " << ( (subEvtWTag != evtWTag) ? 0 : 1)
+		<< ", DTC_CHECK = " << ( ((dtcs_.insert(dtcID).second) && (!isFirstEvent_)) ? 0 : 1);
+	    metricMan->sendMetric("SubEventSnapshot", oss.str(), "xml_string",
```

Comment:
Again, consider using numeric metrics. Metric collection is very fast, so don't worry about sending multiple metrics.

Reply:
done